app/js/questions.js

```diff
@@ -9,26 +8,27 @@ function fillUserTable(HBTemplate) {
     let baseUrl = getBaseUrl()
     let template = Handlebars.compile(HBTemplate)
     let counter = 0;
-    fetch(baseUrl + "question")
+    let cookie = getCookie ('userEmail');
+    getData(`user?email=${cookie}`)
+        .then((data) => {fetch(`${baseUrl}question?test_id=${data.data.test_id}`)
         .then(function(result) {
-            return result.json()
+        return result.json()
+    }).then(function(result) {
```

Comment:
Happy with the new formatting, however, the first .then uses a fat arrow, and the second uses a normal anon function. One or the other, please!